ios/RNSScreen.mm

```diff
@@ -1305,6 +1300,10 @@ - (void)updateProps:(react::Props::Shared const &)props oldProps:(react::Props::
     [self setReplaceAnimation:[RNSConvert RNSScreenReplaceAnimationFromCppEquivalent:newScreenProps.replaceAnimation]];
   }
 
+#if !TARGET_OS_TV && !TARGET_OS_VISION
+  [self updateFormSheetPresentationStyle];
+#endif // !TARGET_OS_TV && !TARGET_OS_VISION
```

Comment:
I would not clutter up this method.

`finalizeUpdates` if a right place to run such side effects. Just limit calling it to when the props have changed. You can achieve this by checking the udpate mask (you have access to it as a parameter of `finalizeUpdates:` method.

Reply:
@kkafar I'm wondering why dragging the sheet triggers that method? Dunno how you could limit that. It is the size changes that's causing it to trigger?